app/facades/trip_legs_facade.rb

```diff
@@ -0,0 +1,35 @@
+class TripLegsFacade
```

Comment:
I'm not sold on `TripLegsFacade` -- mostly because it seems to suggest it is the output from the `trip_legs` table -- which it isn't.

Maybe our `TripFacade` should become `InitialTripFacade` and this should become `FullTripFacade` -- Something to consider for the future

Reply:
Names are hard.  I like your idea for InitialTripFacade and FullTripFacade. Added as issue #70